### PR TITLE
ISL ctime and mtime labels

### LINKED ISSUE #856: Add "create time" and "last modify time" fields to ISL
To easy tracking of network topology alteration we need at least "create time" and "last modify time" on all persistent objects.

### PR DESCRIPTION
**Pull request by surabujin**
Closes #856

### CODE REVIEW

**Comment by surabujin** [comment]
Due to minor change into messaging (IslInfoData) this change affect a lot of components

File: services/src/messaging/src/main/java/org/openkilda/messaging/info/event/IslInfoData.java [hunk 1]
```diff
@@ -15,31 +15,23 @@
...
 
 import java.util.List;
 import java.util.Objects;
 
 /**
  * Defines the payload payload of a Message representing an isl info.
  */
-@JsonSerialize
+@Data
+@ToString
```
Comment (sergii-iakovenko) [inline services/src/messaging/src/main/java/org/openkilda/messaging/info/event/IslInfoData.java]: @Data already includes @ToString, no need to specify separately.

File: services/src/messaging/src/main/java/org/openkilda/messaging/info/event/IslInfoData.java [hunk 2]
```diff
@@ -119,54 +90,29 @@ public IslInfoData(@JsonProperty("latency_ns") long latency,
...
-     *
-     * @return available bandwidth
-     */
-    public long getAvailableBandwidth() {
-        return availableBandwidth;
+        this.availableBandwidth = availableBandwidth;
+        this.state = state;
+        this.timeCreateMillis = timeCreateMillis;
+        this.timeModifyMillis = timeModifyMillis;
+        this.id = String.format("%s_%s", path.get(0).getSwitchId(), String.valueOf(path.get(0).getPortNo()));
```
Comment (sergii-iakovenko) [inline services/src/messaging/src/main/java/org/openkilda/messaging/info/event/IslInfoData.java]: Use "%s_%d" format here instead of String.valueOf

**Comment by surabujin** [comment]
Put multi-threading restoration patch in separate commit, so it can be easily reverted.

**Pull request merged**